qiskit/algorithms/quantum_time_evolution/variational/principles/real/implementations/real_mc_lachlan_variational_principle.py

```diff
@@ -70,7 +88,7 @@ def _calc_metric_tensor(
     def _calc_evolution_grad(
         raw_evolution_grad_imag: OperatorBase, param_dict: Dict[Parameter, Union[float, complex]]
     ) -> OperatorBase:
-        return raw_evolution_grad_imag.bind_parameters(param_dict)
+        return raw_evolution_grad_imag.bind_parameters(param_dict) / 2.0
```

Comment:
A comment to explain the constant would help.

Reply:
There is also no explanation on the equivalent line 85 for the metric.

Reply:
It's just the variational principle.